openstreetmap-carto.lua

```diff
@@ -203,6 +203,7 @@ local roads_info = {
         primary_link    = {z = 220, roads = true},
         secondary_link  = {z = 210, roads = true},
         tertiary_link   = {z = 200, roads = false},
+        busway          = {z = 180, roads = false},
```

Comment:
Is there a technical reason not to add this? Because I understand that for it to have effect the database reload is needed, but is its presence harmful if merged before that is done? (Which is what you are implying.)

Reply:
If this is a line that is desirable in any case (not doing anything as of yet, providing the desired z-order after a database reload), is there a reason you want it removed now?

Reply:
A z-value in openstreetmap-carto.lua is necessary for introducing a new road class so a database reload is inevitable for this.  What i think @jeisenbe meant to express is that this change would depend on a database reload in any case, not that you can make changes to avoid that.  It just means it makes sense to look a bit ahead and think what other changes should be made that require a database reload, like https://github.com/gravitystorm/openstreetmap-carto/issues/3581#issuecomment-826713757

Reply:
Moved to a separate commit. Can be left out if so desired.